config/settings/local.py

```diff
@@ -119,7 +119,8 @@
     "localhost:8080",
     "0.0.0.0",
 ]
-
+METACI_LONG_RUNNING_BUILD_CLASS = "metaci.build.autoscaling.LocalOneOffBuilder"
+METACI_LONG_RUNNING_BUILD_CONFIG = {}
```

Comment:
This is the default for METACI_LONG_RUNNING_BUILD_CONFIG from base.py; no need to override

Reply:
6795ee55